hedera-node/src/main/java/com/hedera/services/legacy/core/jproto/JEd25519Key.java

```diff
@@ -65,4 +68,21 @@ public boolean isValid() {
 			}
 		}
 	}
+
+	@Override
+	public boolean equals(Object o) {
+		if (o == this) {
+			return true;
+		}
+		if (o == null || JEd25519Key.class != o.getClass()) {
+			return false;
+		}
+		final var that = (JEd25519Key) o;
+		return Arrays.equals(this.ed25519, that.ed25519);
+	}
+
+	@Override
+	public int hashCode() {
+		return Objects.hash(Arrays.hashCode(ed25519));
```

Comment:
`Object.hash()` is redundant. Note that its argument is a boxed integer value.

Reply:
I see that `Objects.hash()` internally does `Arrays.hashCode()`

Reply:
That's right only it's `Arrays.hashCode(Object a[])` and we need `Arrays.hashCode(byte a[])`.